src/OrchardCore/OrchardCore.Email.Core/Services/SmtpService.cs

```diff
@@ -1,327 +0,0 @@
-using System;
-using System.Collections.Generic;
-using System.IO;
-using System.Net.Security;
-using System.Security.Cryptography.X509Certificates;
-using System.Threading;
-using System.Threading.Tasks;
-using MailKit.Net.Proxy;
-using MailKit.Net.Smtp;
-using MailKit.Security;
-using Microsoft.Extensions.Localization;
-using Microsoft.Extensions.Logging;
-using Microsoft.Extensions.Options;
-using MimeKit;
-
-namespace OrchardCore.Email.Services
-{
-    /// <summary>
-    /// Represents a SMTP service that allows to send emails.
-    /// </summary>
-    public class SmtpService : ISmtpService
```

Comment:
This is a binary-breaking change!!

Reply:
Only if you are depending on `SmtpService` directly. Usually `ISmtpService` is injected and used. So if someone in inheriting from `SmtpService` directly, they would have a compile time error. So it is okay.